bundles/framework/myplaces3/view/PlaceForm.js

```diff
@@ -437,7 +439,7 @@ Oskari.clazz.define('Oskari.mapframework.bundle.myplaces3.view.PlaceForm',
          */
         _disableFormSubmit: function () {
             this.defaultProps.formSettings.disabledButtons = true;
-            this._renderForm();
+            this._renderForm(this.container);
```

Comment:
I don't get this. You removed the param handling from renderForm(), but you have added the param here? Toggling values in something called "defaultProps" at runtime seems a bit misleading as well. Maybe rename them as just formProps or have another variable for things that change at runtime and merge defaults and the runtime variable when passing values to the form.